### PR TITLE
Add permissions explanation to import

### LINKED ISSUE #5665: Permissions required to Import data
Users in the default Developers group can’t run the Import command unless fs:ImportFromStorage and fs:CreateMetaRange permissions are given to users (https://treeverseio.slack.com/archives/C050J0JGL7J/p1680152666820979). This information is not documented.

_Originally opened by @kesarwam in the wrong repo._

### PR DESCRIPTION
**Pull request by itaiad200**
closes #5665

### CODE REVIEW

File: docs/howto/import.md [hunk 1]
```diff
@@ -22,6 +22,22 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
...
 
+#### User Permissions
+
+The lakeFS user running the import must have the following permissions:
+`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit` and 
+`fs:ImportFromStorage`. The first 3 permissions are available by default to 
+users in the default Developers group ([RBAC](../reference/rbac.md)) or the 
+Writers group ([ACL](../reference/access-control-lists.md)). The last 
+permission enables the user to import data from any location of the storage 
+provider that lakeFS has access to. Thus, it's only available to users in the
```
Comment (guy-har) [inline docs/howto/import.md:33]: Maybe we should emphasize this sentence. Or even add it as some kind of warning on top

**Review by guy-har — APPROVED** [review]
LGTM 
Thanks

File: docs/howto/import.md [hunk 2]
```diff
@@ -22,6 +22,22 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
...
+#### User Permissions
+
+The lakeFS user running the import must have the following permissions:
+`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit` and 
+`fs:ImportFromStorage`. The first 3 permissions are available by default to 
+users in the default Developers group ([RBAC](../reference/rbac.md)) or the 
+Writers group ([ACL](../reference/access-control-lists.md)). The last 
+permission enables the user to import data from any location of the storage 
+provider that lakeFS has access to. Thus, it's only available to users in the 
+default SuperUsers group ([RBAC](../reference/rbac.md)) or the Super
```
Comment (arielshaqed) [inline docs/howto/import.md:34]: I think that we should rephrase to make it more procedural.  Perhaps something like:

> ... only available to users in group Supers (on ACL) or SuperUsers (on RBAC)

?  And similarly for all other alternatives.

File: docs/howto/import.md [hunk 3]
```diff
@@ -22,6 +22,22 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
...
+The lakeFS user running the import must have the following permissions:
+`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit` and 
+`fs:ImportFromStorage`. The first 3 permissions are available by default to 
+users in the default Developers group ([RBAC](../reference/rbac.md)) or the 
+Writers group ([ACL](../reference/access-control-lists.md)). The last 
+permission enables the user to import data from any location of the storage 
+provider that lakeFS has access to. Thus, it's only available to users in the 
+default SuperUsers group ([RBAC](../reference/rbac.md)) or the Super 
+group ([ACL](../reference/access-control-lists.md)). RBAC installations can 
+add that permission to any group, including Developers.
```
Comment (arielshaqed) [inline docs/howto/import.md:36]:
```suggestion
modify policies to add that permission to any group, such as Developers.
```

File: docs/howto/import.md [hunk 4]
```diff
@@ -22,6 +22,22 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
 
 ### Prerequisites
 
+#### User Permissions
+
+The lakeFS user running the import must have the following permissions:
+`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit` and 
+`fs:ImportFromStorage`. The first 3 permissions are available by default to
```
Comment (arielshaqed) [inline docs/howto/import.md:29]:
```suggestion
To run import you need the following permissions:
`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit`, and 
`fs:ImportFromStorage`. The first 3 permissions are available by default to 
```

(Note the extra "Oxford comma".  I think @YaelRiv had a rule to use those, but of course we can change it.  @rmoff , any preference?

**Review by arielshaqed — COMMENTED** [review]
Neat, thanks!  Cleaned up some language.

**Review by arielshaqed — APPROVED** [review]
AND **FORGOT TO _APPROVE_**.  I had _one_ job...

File: docs/howto/import.md [hunk 5]
```diff
@@ -22,6 +22,22 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
...
+
+To run import you need the following permissions:
+`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit`, and 
+`fs:ImportFromStorage`. The first 3 permissions are available by default to 
+users in the default Developers group ([RBAC](../reference/rbac.md)) or the 
+Writers group ([ACL](../reference/access-control-lists.md)). The last 
+permission enables the user to import data from any location of the storage 
+provider that lakeFS has access to. Thus, it's only available to users in the 
+default SuperUsers group ([RBAC](../reference/rbac.md)) or the Super 
+group ([ACL](../reference/access-control-lists.md)). RBAC installations can
```
Comment (itaiad200) [inline docs/howto/import.md:35]:
```suggestion
provider that lakeFS has access to. Thus, it's only available to users in group Supers ([ACL](../reference/access-control-lists.md)) or SuperUsers ([RBAC](../reference/rbac.md)). RBAC installations can 
```

File: docs/howto/import.md (hunk 2 above)
Comment (itaiad200) [reply docs/howto/import.md]: done.

File: docs/howto/import.md (hunk 1 above)
Comment (itaiad200) [reply docs/howto/import.md]: IMO it's bad enough that we have to start an how-to page with permissions adjustments for some non-trivial use-cases. Adding warnings on top of that gets them further from the value..

**Comment by itaiad200** [comment]
@kesarwam you're right, good catch! I think it should however, so I'm fixing the code not the docs.

**Pull request merged**